src/routes/middleware/validate_subscription_update_input.py

```diff
@@ -0,0 +1,23 @@
+from functools import wraps
+
+from flask import jsonify, request
+from marshmallow import ValidationError
+
+from ...schemas.SubscriptionSchema import SubscriptionSchema
+from ..utils.StatusCode import HttpStatus
+
+
+def validate_subscription_update_input(f):
+    @wraps(f)
+    def decorated_function(*args, **kwargs):
+        schema = SubscriptionSchema(partial=True)
```

Comment:
SubscriptionUpdateSchema

Reply:
i think we fixed that together or am i wrong ?